public/src/components/header.jsx

```diff
@@ -18,6 +18,9 @@ export default class Header extends Component {
         { title: 'Account', href: `/account/${userId}` },
         { title: 'Logout', href: '/logout' }
       ];
+      if (this.props.userInfo && this.props.userInfo.is_admin) {
```

Comment:
Could you elaborate? I added this conditional bc only the admin should be able to view the admin page.